content/apps/using-openid.md

```diff
@@ -0,0 +1,122 @@
+# Using OpenID Connect to authenticate users
+
+::: warning
+**This feature is available for our partners using an Akeneo sandbox environment. It will be available for all soon.**
+:::
+
+::: tips
+This is an optional feature in Apps, you can also use your own Authentication.
+:::
+
+To authenticate the users coming from Akeneo PIM, you can use the OpenID Connect protocol.
+
+::: info
+**What is OpenID Connect?**  
+OpenID Connect is a simple identity layer on top of the OAuth 2.0 protocol.  
+See [OpenID official website](https://openid.net/connect/) for more info.
+:::
+
+Basically, you use the same process as for Authorization, but you request an additional scope
+and you will receive, alongside the Access Token, an ID Token containing the information of the current user.
+
+![App authentication diagram](../img/apps/app-authentication-sequence-diagram.png)
+
+## Authenticate during the first Authorization request
+
+If an user is trying to connect your App for the first time, and you want to authenticate him, 
+ask for OpenID scopes during the [Authorization Request](/apps/using-oauth2.html#authorization-request).
+
+```
+
+https://my-pim.cloud.akeneo.com/connect/apps/v1/authorize?
+    response_type=code&
+    client_id=[OAUTH_CLIENT_ID]&
+    scope=openid email read_products write_products&
+    state=[STATE]
+```
+
+::: info
+You can consult the list of [available authentication scopes](/apps/access-scopes.html#available-authentication-scopes).
+:::
+
+## Authenticate after the first Authorization request
+
+If an user is trying to access your App from his Akeneo PIM, and you want to authenticate him, start an
+[Authorization Request](/apps/using-oauth2.html#authorization-request),
+**even if you already are connected to his Akeneo PIM**.
+
+**During this new Authorization Request, you must request all the scopes your App needs**, including the Authorization scopes,
+in addition of the OpenID scopes.  
+
+## Extracting user information from the ID Token
+
+At the end of the OAuth 2.0 process, if your Access Token Request is accepted, you will receive a JSON response with both tokens:
```

Comment:
note that the `access_token` you'll get with the authentication request, is the same `access_token` you get during the **authorization** workflow.